### PR TITLE
Examples for chapter 11

### PR DESCRIPTION
**Pull request by AndresRPerez12**
Code Examples for the articles in Chapter 11 on the Try Catch Finally block

### CODE REVIEW

File: src/main/java/io/jcoder/tutorials/ch11/CatchMultipleTypes/CatchMultipleTypesApp.java [hunk 1]
```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright 2018 - JCoder Ltd
```
Comment (camilojc) [inline src/main/java/io/jcoder/tutorials/ch11/CatchMultipleTypes/CatchMultipleTypesApp.java]: minor - please update the Copyright year :)

File: src/main/java/io/jcoder/tutorials/ch11/CatchMultipleTypes/CatchMultipleTypesApp.java [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+ */
+package io.jcoder.tutorials.ch11.CatchMultipleTypes;
```
Comment (camilojc) [inline src/main/java/io/jcoder/tutorials/ch11/CatchMultipleTypes/CatchMultipleTypesApp.java]: package names should be lowercased, can you rename to `io.jcoder.tutorials.ch11.multiplecatch` for example please? :)

File: src/main/java/io/jcoder/tutorials/ch11/CatchMultipleTypes/CatchMultipleTypesApp.java (hunk 1 above)
Comment (AndresRPerez12) [reply src/main/java/io/jcoder/tutorials/ch11/CatchMultipleTypes/CatchMultipleTypesApp.java]: Fixed it and pushed the corrected version to my branch.

File: src/main/java/io/jcoder/tutorials/ch11/CatchMultipleTypes/CatchMultipleTypesApp.java (hunk 2 above)
Comment (AndresRPerez12) [reply src/main/java/io/jcoder/tutorials/ch11/CatchMultipleTypes/CatchMultipleTypesApp.java]: Fixed it and pushed the corrected version to my branch.

**Review by camilojc — APPROVED** [review]

**Pull request merged**